packages/oui-checkbox/src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe("ouiCheckbox", () => {
                 };
 
                 const element = TestUtils.compileTemplate("<oui-checkbox model=\"$ctrl.currentModel\"></oui-checkbox>", context);
-                const $ctrl = TestUtils.getElementController(element);
+                const $ctrl = TestUtils.constructor.getElementController(element);
```

Comment:
Very ugly. VERY I mean.  An exception should be made for this case.

Reply:
Do you prefer that I export the `TestUtils` class and I import it under `TestUtilsClass` so I could do `TestUtilsClass.getElementController(element)` instead ?

Reply:
I applied the eslint-disable comment, but if it happens too much we need to find a better solution.